samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
std::array is more readable, but this has less dependencies. Returning a reference to an array requires no additional dependencies for users that do not care about this feature.